modin/logging/config.py

```diff
@@ -0,0 +1,100 @@
+import logging
+import datetime as dt
+import os
+import uuid
+import platform
+import psutil
+import pkg_resources
+import threading
+import time
+from modin.config import LogMode
+
+__LOGGER_CONFIGURED__: bool = False
+
+
+class MyFormatter(logging.Formatter):
+    converter = dt.datetime.fromtimestamp
+
+    def formatTime(self, record, datefmt=None):
+        ct = self.converter(record.created)
+        if datefmt:
+            s = ct.strftime(datefmt)
+        else:
+            t = ct.strftime("%Y-%m-%d %H:%M:%S")
+            s = "%s,%03d" % (t, record.msecs)
```

Comment:
It's hard to tell what these lines are doing. Could you add more descriptive names or comments?